doc/sphinx/ref/azure.core.tracing.rst

```diff
@@ -1,12 +1,6 @@
 azure.core.tracing package
 ==========================
 
-Subpackages
------------
-
-.. toctree::
-
-   azure.core.tracing.ext
```

Comment:
Do you want to add azure.core.tracing.opencensus here?

Reply:
Nope, azure-core should be "plugin agnostic" and not mention it.

Reply:
We might want to update the Readme to mention azure-core-tracing-opencensus though, to give a pointer to the package.